features/testing.feature

```diff
@@ -23,3 +23,21 @@ Feature: Test that WP-CLI loads.
       """
       DISABLE_WP_CRON is: true
       """
+
+  @require-wp-6.0 @require-sqlite
```

Comment:
`@require-wp-6.0` shouldn't be needed if we disable SQLite + WP 6.0 combo in the test matrix itself. Or what was the reason for adding it now?

If the matrix updates (which we need to do in a separate PR in the `wp-cli/.github` repo) don't suffice, we could do something like this in `behat-tags.php` instead:

```php
if ( 'sqlite' !== getenv( 'DB_TYPE' ) || version_compare( getenv( 'WP_VERSION' ), '6.0', '<' ) ) {
	$skip_tags[] = '@require-sqlite';
}
```

But I don't think it's needed right now.

Reply:
makes sense, removed this